packages/react-conformance/src/utils/consoleUtil.ts

```diff
@@ -8,12 +8,16 @@ let isDisabledOnce: boolean;
 /**
  * Enable console logging.
  */
-export const enable = () => Object.assign(console, original);
+export const enable = () => {
+  Object.assign(console, original);
+};
 
 /**
  * Disable console logging.
  */
-export const disable = () => Object.assign(console, disabled);
+export const disable = () => {
+  Object.assign(console, disabled);
+};
```

Comment:
I don't quite understand why these utils exist

Reply:
Apparently for this
```ts
    it(`renders component as HTML tags or passes "as" to the next component`, () => {
      try {
        // silence element nesting warnings
        consoleUtil.disableOnce();
```